src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,210 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import StarRatings from "react-star-ratings";
+import { Navbar } from "../../components/components-index";
+import { useAppContext } from "../../context/AppContext";
+import { Link } from "react-router-dom";
+import "./Cart.css";
+
+export const Cart = () => {
+  const encodedToken = localStorage.getItem("token");
+  axios.defaults.headers.common["authorization"] = encodedToken;
+
+  const { appState, appDispatch } = useAppContext();
+  const [cartItemValue, setCartItemValue] = useState(0);
+
+  useEffect(() => {
+    (async () => {
+      setCartItemValue(appState.cartItems);
+      try {
+        const response = await axios.get("/api/user/cart");
+        response && appDispatch({ type: "CART", payload: response.data.cart });
```

Comment:
dispatch type can be more descriptive than "CART"

Reply:
Done!

Reply:
Done!